1957/CORRECTED/00000005.txt

```diff
@@ -1,14 +1,14 @@
-ANNUAL SACKED HARP SINGINGS
-3
 Fourth—Rock Creek Church, 5 Miles N. E. of Double Springs.
-Fourth—Sullins School House, Marion County, on Hackleburg and Hamil¬
+Fourth—Sullins School House, Marion County, on Hackleburg and Hamil-
 ton Road.
 Fourth, and Saturday before—Courthouse at Guntersville.
+

```

Comment:
Are you sure we want to add spaces? @jpkarlsberg what do you think

Reply:
```suggestion
```